wpilibc/src/main/native/include/frc/Encoder.h

```diff
@@ -81,7 +54,7 @@ class Encoder : public ErrorBase,
    *                         be double (2x) the spec'd count.
    */
   Encoder(int aChannel, int bChannel, bool reverseDirection = false,
```

Comment:
Use WPI_DEPRECATED() on the constructors so a warning message to users is generated.

Reply:
How to do this / is there a class to imitate on this?

Reply:
Like this: https://github.com/wpilibsuite/allwpilib/blob/471f375a389d458ce1466ff65a8abb9b6cf332da/wpilibc/src/main/native/include/frc/smartdashboard/SendableHelper.h#L46-L55

Reply:
thanks, will do now!

Reply:
Is it ok now?